normandy/recipes/tests/test_api.py

```diff
@@ -101,9 +101,15 @@ def test_it_cant_edit_actions_in_use(self, api_client, settings):
 
         res = api_client.patch('/api/v1/action/active/', {'implementation': 'foobar'})
         assert res.status_code == 403
+        assert res.data == {
+            'detail': 'Cannot edit this object while it is in use'
```

Comment:
I want to assert that the right kind of error got thrown. I want to make sure this is an "object in use" error, and not an "admin disabled" error. We could also this with something like `assert res.data['detail'] == NotInUse.message`, if changing error strings ends up being annoying.